test/kwc-feature-list_test.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>kwc-feature-list test</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="../kwc-feature-list.html">
+    </head>
+    <body>
+        <test-fixture id="basic">
+            <template>
+                <kwc-feature-list></kwc-feature-list>
+            </template>
+        </test-fixture>
+
+        <script>
+            suite('kwc-feature-list', function() {
+                var element;
+
+                setup(function () {
+                    element = fixture('basic');
+                });
+
+                test('instantiating the element works', function() {
+                    assert.equal(element.is, 'kwc-feature-list');
+                });
+
+                test('the title is undefined by default', function() {
+                    assert.equal(element.title, undefined);
+                });
+
+                test('the title can be set imperatively', function(done) {
+                    element.title = 'Features'
+                    assert.equal(element.title, 'Features');
+                    flush(function() {
+                        var title =  Polymer.dom(element.root).querySelectorAll('h2')[0];
+                        assert.equal(title.textContent, 'Features');
+                        done();
+                    });
+                });
+
+                test('the features are an empty array by default', function() {
+                    assert.isArray(element.features);
+                });
+
+                test('the features can be set imperatively', function(done) {
+                    var imageUrl = 'https://placekitten.com/300/150';
```

Comment:
I would recommend us to save those parameters we are setting on different variables so when comparing values, length and other properties we don't need to compare with literal values.